platform/src/main/java/com/yurjinia/platform/common/utils/PayloadUtils.java

```diff
@@ -0,0 +1,49 @@
+package com.yurjinia.platform.common.utils;
+
+import com.yurjinia.platform.auth.controller.request.RegistrationRequest;
+import com.yurjinia.platform.common.content.ContentGeneratorService;
+import com.yurjinia.platform.project_structure.board.controller.request.CreateBoardRequest;
+import com.yurjinia.platform.project_structure.column.controller.request.CreateColumnRequest;
+import com.yurjinia.platform.project_structure.project.dto.CreateProjectRequest;
+import lombok.experimental.UtilityClass;
+
+import java.util.HashSet;
+
+@UtilityClass
+public class PayloadUtils {
+
+    private final ContentGeneratorService contentGeneratorService = new ContentGeneratorService();
```

Comment:
no bean dependencies must be in utils class, instead where you use the service provide the result of generated value to method input.